package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-chromecast",
-  "version": "1.0.15",
+  "version": "1.0.16",
```

Comment:
Should really leave this version bump out of the bugfix commit.  Then after the PR is merged we can 

```bash
npm version patch && npm publish
```

Which will bump the version, publish and create some nice git tags for us